public/app/core/directives/ng_model_on_blur.ts

```diff
@@ -47,7 +47,7 @@ function validTimeSpan() {
         if (viewValue.indexOf('$') === 0 || viewValue.indexOf('+$') === 0) {
           return true; // allow template variable
         }
-        var info = rangeUtil.describeTextRange(viewValue);
+        var info = rangeUtil.describeTextRange(viewValue.replace(/^-/, ''));
```

Comment:
why remove minus sign?

Reply:
It is needed; otherwise describeTextRange says the interval is invalid.

Reply:
but that just means we need to update describeTextRange yes?